### ISSUE TITLE
Developer cannot be verified on macOS

### ISSUE THREAD

**Issue by hussa6** [Studio3T/robomongo#1840]
<img width="372" alt="Screenshot 2021-06-16 at 17 12 48" src="https://user-images.githubusercontent.com/59232894/122255553-25ebf800-cec6-11eb-87b5-4e2aea7a1184.png">

Seems like it hasn't been code signed and notarised for macOS? Any reason for this?

**Comment by jowpurcinelli** [comment]
Is that the first time you're opening it?
Just open it in Finder. Right-click the app and click Open. 
Mac does that for a lot of apps that we're downloaded from "untrusted" sources.

**Comment by hussa6** [comment]
Yep. It's doing that because it hasn't been code-signed (notarized on macOS) hence the developer is unverified (untrusted on the OS).
